.github/Security_issue.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Security Issue
+about: Report security-related errors
+
+---
+
+<!--
+    The Symfony Code of Conduct applies to all the activity on this repository.
+    See https://symfony.com/doc/current/contributing/code_of_conduct/index.html
+-->
+
+If you have found a security issue in Symfony, please send the details to
+security [at] symfony.com and don't disclose it publicly until we can provide a
```

Comment:
do we have a security contact? i think there is no official one, maybe ask them to email you and me?